[14.0][ADD] mrp_subcontracting_bom_dual_use: New addon. (#834)
OCA/manufacture: mrp_subcontracting_bom_dual_use/readme/USAGE.rst

> * Go to *Manufacturing > Products > Bill of materials* and create new subcontracting record and set Allow in reguilar production chekbox.

Reviewer: ```suggestion
* Go to *Manufacturing > Products > Bill of materials* and create new subcontracting record and set Allow in regular production checkbox.
```

Reviewer: Anyways, this check should be only shown in case of a subcontracting BoM.

> * Try to manufacture the product and you will be able to use the previously created bill of materials.

Reviewer: ```suggestion
* Try to manufacture the product, and you will be able to use the previously created bill of materials.
```

> * Go to *Manufacturing > Products > Bill of materials* and create new subcontracting record and set Allow in regular production checkbox.
> * Try to manufacture the product, and you will be able to use the previously created bill of materials.

Reviewer: ```suggestion
#. Go to *Manufacturing > Products > Bill of materials* and create a new one with type = "Subcontracting".
#. Check "Allow in regular production".
#. Try to manufacture the product, and you will be able to use the previously created bill of materials.
```
